cmd/collectors/zapi/plugins/shelf/shelf.go

```diff
@@ -149,8 +150,7 @@ func (my *Shelf) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 	if x := result.GetChildS("attributes-list"); x != nil {
 		shelves = x.GetChildren()
```

Comment:
can we wrap other code in handlecmode() ?

Reply:
done